src/tigerbeetle/main.zig

```diff
@@ -446,6 +447,110 @@ const Command = struct {
             watchdog.detach();
         }
 
+        if (!args.development) {
+            // Try to lock all memory in the process to avoid the kernel swapping pages to disk and
+            // potentially introducting undetectable disk corruption into memory.
+            // This is a best-effort attempt and not a hard rule as it may not cover all memory edge
+            // case. So warn on error to notify the operator to adjust conditions if possible.
+            const lock_mem_err = "Unable to lock pages in memory ({s})" ++
```

Comment:
```suggestion
            const mlockall_error = "Unable to lock pages in memory ({s})" ++
```

Per TigerStyle, don't abbreviate variable names. For example, write `error` not `err`.

Also, relate the error to the syscall, e.g. `mlockall`